staging/cse/windows/provisioningscripts/loggenerator.ps1

```diff
@@ -87,6 +87,15 @@ $kLogFiles | Foreach-Object {
     Create-SymbolLinkFile -SrcFile (Join-Path "C:\k\" $_) -DestFile (Join-Path $aksLogFolder $_)
 }
 
+$nvidiaInstallLogFolder="C:\AzureData\NvidiaInstallLog"
+if (Test-Path $nvidiaInstallLogFolder) {
+    $logFiles=Get-ChildItem (Join-Path $nvidiaInstallLogFolder *.log)
+    $logFiles | Foreach-Object {
+        $fileName = [IO.Path]::GetFileName($_)
+        Create-SymbolLinkFile -SrcFile (Join-Path $nvidiaInstallLogFolder $fileName) -DestFile (Join-Path $aksLogFolder $fileName)

```

Comment:
Does the gpu install log file have a unique prefix so there is no conflict when creating symbol links in $aksLogFolder?

Reply:
Yes, they all have the prefix "LOG."
<img width="293" alt="image" src="https://github.com/Azure/AgentBaker/assets/44663669/6569d77b-78e4-475a-b10f-0d6ca8b65619">